### PR TITLE
omas: Add omas recipe

### PR DESCRIPTION
**Pull request by smithsp**

### CODE REVIEW

**Comment by bgruening** [comment]
@smithsp `Oma` means in German `grandmother` ... `Omas recipes` is just too good! Made my day.

File: recipes/omas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+  patches:
+    - setup.py.patch
```
Comment (synapticarbors) [inline recipes/omas/meta.yaml]: It's unclear to me that this patch is required as `--single-version-externally-managed` ignores `install_requires`

File: recipes/omas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - numpy
+    - netcdf4
+    - boto3
+    - botocore 1.7.*
+
+about:
+  home: https://gafusion.github.io/omas
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```
Comment (synapticarbors) [inline recipes/omas/meta.yaml]: Please raise an issue upstream about including the `license_file` in the source distribution and put a link to the issue (or PR) as a comment in this recipe as a reminder to look for its inclusion in future releases.

**Comment by smithsp** [comment]
@bgruening Glad to hear you enjoyed your Oma's recipes.  (Mention @orso82)

**Pull request merged**